inc/opx/std_ip_utils.h

```diff
@@ -31,6 +31,7 @@
 //! TODO move the types from db_common here and redefine them in the db-api to use these
 #include "ds_common_types.h"
 #include <netinet/in.h>
+#include <arpa/inet.h>
```

Comment:
Is this header file included for htonl()? 
With the previous version of std_ip_utils.h, we did use htonl() but dint include this header. Wondering how did it even compile then before? Because it was pre-processed, not compiled?

Reply:
I'm not sure what you mean 'previous version' of std_ip_utils.h we used htonl(). My changes is the first time this function is used in this header-file. In any case, we want to explicitly include header-files that has functions we use because we want to be able to compile stand-alone header-files.